src/app/pages/login/login.component.html

```diff
@@ -72,7 +72,7 @@ <h1 class="text-white">I Stand With Health Professionals.?</h1>
           <a href="javascript:void(0)" class="text-light"><small>Forgot password?</small></a>
         </div>
         <div class="col-6 text-right">
-          <a href="javascript:void(0)" class="text-light"><small>Create new account</small></a>
+          <a href="/#/register" class="text-light"><small>Create new account</small></a>
```

Comment:
Please take a look at how register link is working on auth-layout.component.html. Whenver possible we should use the router than the hard coded href value. Your will work but that is not the right approach.